src/OpenSage.Game/OpenSage.Game.csproj

```diff
@@ -6,17 +6,17 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.7.0" />
     <PackageReference Include="MoonSharp" Version="2.0.0" />
     <PackageReference Include="SixLabors.ImageSharp" Version="1.0.0-beta0006" />
     <PackageReference Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta0006" />
     <PackageReference Include="SharpAudio" Version="$(SharpAudioVersion)" />
     <PackageReference Include="SharpAudio.Util" Version="$(SharpAudioVersion)" />
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.1" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.7.0" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Veldrid" Version="$(VeldridVersion)" />
     <PackageReference Include="Veldrid.ImageSharp" Version="$(VeldridVersion)" />
-    <PackageReference Include="Veldrid.ImGui" Version="$(ImGuiVersion)" />
+    <PackageReference Include="Veldrid.ImGui" Version="5.72.0" />
```

Comment:
Are you intentionally not using the variable anymore?

Reply:
Ops, no my bad. I was using the Visual Studio NuGet updater. Will fix this.